lib/handlers/invite/newUserInvitationV2.js

```diff
@@ -25,12 +25,12 @@ const convertMarkdownToHtml = (content) => {
 
 const process = async (config, logger, data) => {
   try {
-    const serviceName = data.serviceName || 'DfE Sign-in';
+    const serviceName = data.serviceName;
```

Comment:
please consider creating newUserInvitation3
. As this will impact all DSI

Reply:
The requirement is for this change to affect all invitations from DfE Sign-In, not just ones from the data migration solution.